cmd/insprd/memory/interface.go

```diff
@@ -97,3 +99,12 @@ type TransactionInterface interface {
 	InitTransaction()
 	Cancel()
 }
+
+// BrokerInterface is the interface tht allows for interaction
+// with the systems multiple brokers
+type BrokerInterface interface {
+	GetAll() utils.StringArray
+	GetDefault() string
+	Create(broker string, config interface{}) error
+	SetDefault(broker string) error
+}
```

Comment:
I know it seems stupid, but redeclare the types as like
```go
type BrokerConfiguration interface{}
type BrokerStatus string
type BrokerStatusArray utils.StringArray
```

It seems useless but it makes it much easier to update the typing in the future, just so we don't need to change every function header and return type.